app/assets/javascripts/admin/addon/components/site-settings/string.hbs

```diff
@@ -1,7 +1,12 @@
 {{#if this.setting.textarea}}
   <Textarea @value={{this.value}} class="input-setting-textarea" />
 {{else if this.isSecret}}
-  <Input @type="password" @value={{this.value}} class="input-setting-string" />
+  <Input
+    @type="password"
+    @value={{this.value}}
+    class="input-setting-string"
+    autocomplete="new-password"
```

Comment:
This seems to help...for now. Chrome kept autofilling "passwords" for me :grimacing: It was happening in the AI plugin too, and on Meta